.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Create a report to help us improve our plugin
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Bug description**
```

Comment:
Use `#` (as many times as required) instead of `**`.
Headers indicate sections more clearly.

Ref: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet

Reply:
Ok, I just adapted template proposed by GH (they use `**` here)